recipe/meta.yaml

```diff
@@ -1,14 +1,13 @@
-{% set name = "dask-ml" %}
-{% set version = "2023.3.24" %}
+{% set name = "dask_ml" %}
+{% set version = "2024.4.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Or just hardcode the name in the URL.

Reply:
@JeanChristopheMorinPerso I was trying to avoid hardcoding the entire URL which lead to the slip up. Thanks @skupr-anaconda I will add your change :)

Reply:
If you ask me, I see no harm in hardcoding the name in the URL.